modules/services/views/src/main/java/au/com/shiftyjelly/pocketcasts/views/activity/WebViewActivity.kt

```diff
@@ -115,11 +115,6 @@ class WebViewActivity : AppCompatActivity(), CoroutineScope {
         }
     }
 
-    private fun uriIsInternal(url: Uri): Boolean {
-        val host = url.host ?: return false
-        return INTERNAL_HOSTS.count { host.endsWith(it) } > 0
-    }
```

Comment:
For more context see [this PR](https://github.com/Automattic/pocket-casts-android-private/issues/2558). This is most likely no longer necessary since we don't support that feature.